src/dispatch/static/dispatch/src/service/ServiceSelect.vue

```diff
@@ -1,23 +1,34 @@
 <template>
-  <v-autocomplete
-    v-model="service"
-    :items="items"
-    :search-input.sync="search"
-    :menu-props="{ maxHeight: '400' }"
-    item-text="name"
-    :label="label"
-    placeholder="Start typing to Search"
-    return-object
-    :hint="hint"
-    :loading="loading"
-    no-filter
-  >
-  </v-autocomplete>
+  <ValidationProvider name="service" immediate>
+    <v-autocomplete
+      v-model="service"
+      :items="items"
+      :search-input.sync="search"
+      :menu-props="{ maxHeight: '400' }"
+      item-text="name"
+      :label="label"
+      placeholder="Start typing to Search"
+      return-object
+      :hint="hint"
```

Comment:
We should remove the hint or update its string. If we update it, we may want to add one to the ValidationProvider document as well.